orttraining/orttraining/python/ort_trainer.py

```diff
@@ -326,11 +326,26 @@ def convert_model_loss_fn_to_onnx(model, loss_fn, model_desc, device, inputs, op
                        do_constant_folding=False,
                        **other_export_options)
 
-    model = onnx.load_model_from_string(f.getvalue())
-
-    model = FuseSofmaxNLLToSoftmaxCE(model)
+    onnx_model = onnx.load_model_from_string(f.getvalue())
+
+    # Remove 'model_.' prefix introduced by model wrapper for initializers.
```

Comment:
Is it necessary because of `class WrapModel`? Is there a way to prevent this "model_." to be added in the first place?

Reply:
"model_" added due to the original model being an attribute "model_" of `WrapModel`. To prevent this we need to avoid creating this extra layer of module. Maybe instead directly overwrite the forward method of original model, seems more hacky as the original forward might be called somewhere else..

Reply:
Good point. We should handle the case where both WrapModel and loss_fn are used. Currently the case is not valid due to some issues with WrapModel. Will fix it in a separate PR.